core/server/data/schema/fixtures/fixtures.json

```diff
@@ -24,7 +24,8 @@
             "entries": [
                 {
                     "name":             "Default Newsletter",
-                    "slug":             "default-newsletter"
+                    "slug":             "default-newsletter",
+                    "show_header_name": false
```

Comment:
How come this is false in the fixture, but true everywhere else? Is it that the current default has it false? I assume so but if so that detail is very much a useful "why" for the commit 😬

Reply:
This is from the issue:
> default for first newsletter automatically created = off, default for any new newsletter added manually = on

This is because by default the newsletter name is the same as the site title, and the site title is already shown by default. I'll add this to the commit